features/step_definitions/board_steps.rb

```diff
@@ -0,0 +1,35 @@
+require 'cucumber/rspec/doubles'
+
+require_relative '../../lib/cli'
+require_relative '../../lib/tic_tac_toe'
+require_relative '../../lib/game_state'
+require_relative '../../lib/game_messenger'
+
+original_stdout = $stdout
+
+When(/^the game starts$/) do
```

Comment:
this step seems to be the same as `When(/^I start the game$/)`
is it possible to reuse steps to avoid duplication

Reply:
I combined the two tests, do you think it looks better now?